frontend/src/components/Button/ButtonStyles.js

```diff
@@ -3,14 +3,14 @@ import { StyleSheet } from "react-native";
 export const styles = StyleSheet.create({
   container: {
     elevation: 8,
-    backgroundColor: "#2E8B57",
```

Comment:
I think it is probably best to revert all changes to the button component for now. It was not intended as a part of this ticket to add any buttons as they will need to be displayed by context. Buttons will always need to be different sizes, colours, etc. so we will have to look at how to pass these in as props to this component.